src/Contracts/MessageTypes.ts

```diff
@@ -37,10 +43,9 @@ export enum MessageTypes {
   DisplayNPSSurvey,
   OpenVCoreMongoNetworkingBlade,
   OpenVCoreMongoConnectionStringsBlade,
-
-  // Data Explorer -> Fabric communication
-  GetAuthorizationToken,
-  GetAllResourceTokens,
+  GetAuthorizationToken, // Data Explorer -> Fabric
```

Comment:
Laurent - can we plan to split message types into Portal message types and Fabric message types in the near future ?

Reply:
The constraint is that we are leveraging the existing messaging infrastructure and its an enum (ie Fabric receives a number), so if we have a completely separate enum file for Fabric, we could get collisions.
Let's take this offline.